pkg/tuned/cmd/render/render.go

```diff
@@ -0,0 +1,251 @@
+	}
+
+	// Get pools, mConfigs and profile from inputDir
+	// Read asset directory fileInfo
+	filePaths, err := util.ListFiles(inputDir)
+	klog.V(4).Infof("listed files: %v", filePaths)
+	if err != nil {
+		return err
+	}
+	// Make output dir if not present
+	err = os.MkdirAll(outputDir, os.ModePerm)
+	if err != nil {
+		return err
+	}
+
+	var (
+		perfProfiles []*performancev2.PerformanceProfile
+		mcPools      []*mcfgv1.MachineConfigPool
+		mcConfigs    []*mcfgv1.MachineConfig
+		ctrcfgs      []*mcfgv1.ContainerRuntimeConfig
+		tuneD        []*tunedv1.Tuned
+	)
+
+	// Iterate through the file paths and read in desired files
+	for _, path := range filePaths {
+		file, err := os.Open(path)
+		if err != nil {
+			return fmt.Errorf("error opening %s: %w", file.Name(), err)
+		}
+		defer file.Close()
+
+		manifests, err := util.ParseManifests(file.Name(), file)
+		if err != nil {
+			return fmt.Errorf("error parsing manifests from %s: %w", file.Name(), err)
+		}
+
+		// Decode manifest files
+		for idx, m := range manifests {
+			obji, err := runtime.Decode(runtimeDecoder, m.Raw)
+			if err != nil {
+				if runtime.IsNotRegisteredError(err) {
+					klog.V(4).Infof("skipping path %q [%d] manifest because it is not part of expected api group: %v", file.Name(), idx+1, err)
+					continue
+				}
+				return fmt.Errorf("error parsing %q [%d] manifest: %w", file.Name(), idx+1, err)
+			}
+
+			switch obj := obji.(type) {
+			case *performancev2.PerformanceProfile:
+				perfProfiles = append(perfProfiles, obj)
+			case *mcfgv1.MachineConfigPool:
+				mcPools = append(mcPools, obj)
+			case *mcfgv1.MachineConfig:
+				mcConfigs = append(mcConfigs, obj)
+			case *mcfgv1.ContainerRuntimeConfig:
+				ctrcfgs = append(ctrcfgs, obj)
+			case *tunedv1.Tuned:
+				tuneD = append(tuneD, obj)
+			default:
+				klog.Infof("skipping %q [%d] manifest because of unhandled %T", file.Name(), idx+1, obji)
+			}
+		}
+	}
+
+	if len(perfProfiles) != 1 {
+		klog.Errorf("%d performance profiles were found. Expecting One", len(perfProfiles))
+		return fmt.Errorf("%d performance profiles were found. Expecting One", len(perfProfiles))
+	}
+
+	profile := perfProfiles[0]
+	tunedFromPP, err := tuned.NewNodePerformance(profile)
+	if err != nil {
+		klog.Error("Unable to get tuned. error : %v", err)
+		return fmt.Errorf("unable to get tuned from profile: %w", err)
+	}
+
+	// add tuned from PP to the list
+	tuneD = append(tuneD, tunedFromPP)
+
+	//extract all the profiles.
+	for _, tt := range tuneD {
+		//REVIEW - Not sure about the "recommended"profile.
+		_, _, _, err = tunedpkg.ProfilesExtract(tt.Spec.Profile, *tunedFromPP.Spec.Recommend[0].Profile)
+		if err != nil {
+			klog.Error("error extracting tuned profiles : %v", err)
+			return fmt.Errorf("error extracting tuned profiles: %w", err)
+		}
+	}
+
+	//Should run tuned
+	tunedrun.TunedOperandRun()
```

Comment:
Not fully sure about this. 
Need to just run `/var/lib/tuned/bin/run start` so maybe something like [execCmd](https://github.com/openshift/cluster-node-tuning-operator/blob/12b9a12c1cf6892b36464c26d6198d2211f28a14/pkg/tuned/tuned_parser.go#L215) could be enough